testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/cors/AbstractCorsExampleAdapterTest.java

```diff
@@ -117,12 +119,32 @@ public void angularCorsProductTest() {
         angularCorsProductPage.loadPublicRealmInfo();
         waitUntilElement(angularCorsProductPage.getOutput()).text().contains("Realm name: cors");
 
-        Client client = ClientBuilder.newClient();
-        WebTarget target = client.target(authServerPage.createUriBuilder()).path("version");
-        VersionRepresentation version = target.request().get(VersionRepresentation.class);
+        String serverVersion = getAuthServerVersion();
 
+        assertNotNull(serverVersion);
+
+        angularCorsProductPage.navigateTo();
+        WaitUtils.pause(2000);
```

Comment:
We shouldn't use those hardcoded waits, I thing waitUntilElement should do its job here. Have you tried it without it?

Reply:
Yeah, it does not work. Not sure why.